activerecord/lib/active_record/relation/record_fetch_warning.rb

```diff
@@ -0,0 +1,50 @@
+module ActiveRecord
+  class Relation
+    module RecordFetchWarning
+      # When this module is prepended to ActiveRecord::Relation and
+      # `config.active_record.warn_on_records_fetched_greater_than` is
+      # set to an integer, if the number of records a query returns is
+      # greater than the value of `warn_on_records_fetched_greater_than`,
+      # a warning is logged. This allows for the dection of queries that
```

Comment:
This change was merged a while ago and after looking at the master branch I see it's already fixed: https://github.com/rails/rails/commit/fe6de0f023ba2632cb2c0ee3e85b59b0eb1b353a :grin: